packages/calcite-components/src/components/tile/tile.tsx

```diff
@@ -8,11 +8,12 @@ import {
 } from "../../utils/interactive";
 import { CSS, SLOTS } from "./resources";
 import { Alignment, Scale } from "../interfaces";
-import { slotChangeHasAssignedElement } from "../../utils/dom";
 
 /**
- * @slot content-start - A slot for adding non-actionable elements before the component's content.
- * @slot content-end - A slot for adding non-actionable elements after the component's content.
+ * @slot content-top - A slot for adding non-actionable elements above the component's content.  Content slotted here will render in place of the `icon` property.
+ * @slot content-bottom - A slot for adding non-actionable elements below the component's content.
+ * @slot content-start - A slot for adding non-actionable elements before the component's content. @deprecated use `content-top` slot instead
```

Comment:
I _think_ `@deprecated` won't render as expected in the generated doc. cc @macandcheese Can you follow [this](https://github.com/Esri/calcite-design-system/blob/007dd64c9da4dc6bad73fb1f9822592bad1c4cd3/packages/calcite-components/src/components/card/card.tsx#L46) deprecation pattern for now?

Reply:
Correct, the linked format for Card will render correctly - https://developers.arcgis.com/calcite-design-system/components/card/#slots

Reply:
Ah, thank you!  I was noticing weird stuff like this in VS Code too using the `@deprecated` syntax.  TS thinks I'm trying to deprecate the whole component, lol:

![image](https://github.com/Esri/calcite-design-system/assets/821864/d25e0b38-5462-4b1e-b45b-e38d6a53e204)